database_admin/migrations/026_packages.up.sql

```diff
@@ -13,16 +13,25 @@ GRANT SELECT, USAGE ON SEQUENCE package_name_id_seq TO evaluator;
 GRANT SELECT, USAGE ON SEQUENCE package_name_id_seq TO listener;
 GRANT SELECT, USAGE ON SEQUENCE package_name_id_seq TO vmaas_sync;
 
+CREATE TABLE IF NOT EXISTS strings
+(
+    id    BYTEA NOT NULL PRIMARY KEY,
+    value TEXT  NOT NULL
+);
+
+GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE strings TO vmaas_sync;
+GRANT SELECT ON TABLE strings TO evaluator;
+GRANT SELECT ON TABLE strings TO listener;
+GRANT SELECT ON TABLE strings TO manager;
+
 CREATE TABLE IF NOT EXISTS package
 (
-    id          INT GENERATED BY DEFAULT AS IDENTITY NOT NULL PRIMARY KEY,
+    id               INT GENERATED BY DEFAULT AS IDENTITY NOT NULL PRIMARY KEY,
     -- TODO: inline vs extract ?
-    name_id     INT                                  NOT NULL REFERENCES package_name,
-    evra        TEXT                                 NOT NULL CHECK (NOT empty(evra)),
-    -- TODO: Inline vs extract to table ?
-    description TEXT                                 NOT NULL CHECK (NOT empty(description)),
-    -- TODO: Inline vs extract to table ?
-    summary     TEXT                                 NOT NULL CHECK (NOT empty(summary)),
+    name_id          INT                                  NOT NULL REFERENCES package_name,
```

Comment:
Isn't "TODO" useless here now?

Reply:
If it's for easy pkg-name-based queries, we can comment it here, or in go struct definition and we can remove this TODO because it's no longer a question.